pythonforandroid/recipes/openssl/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 
 class OpenSSLRecipe(Recipe):
-    version = '1.0.2f'
+    version = '1.0.2g'
```

Comment:
This change has already been made in master, and required a change to the patch as well - but you haven't modified the patch. This can't actually be working as the patch will fail to apply to 1.0.2g and therefore the build will fail.